integrations/facebook-pixel/test/index.test.js

```diff
@@ -2748,9 +2751,10 @@ describe('Facebook Pixel', function() {
 
   describe('#formatTraits', function() {
     it('should return an empty object if traits is falsy', function() {
-      var expected = {};
-
       analytics.reset();
+
+      var userData = analytics.user();
+      var expected = { external_id: userData.anonymousId() };
```

Comment:
`analytics.user().anonymousId()` should work here and would be one less line of code. Just a style suggestion. :)